lib/extractors/text.js

```diff
@@ -10,8 +10,12 @@ function extractText( filePath, options, cb ) {
       cb( error, null );
       return;
     }
-    encoding = jschardet.detect( data ).encoding.toLowerCase();
-    decoded = iconv.decode( data, encoding );
+    try {
+      encoding = jschardet.detect( data ).encoding.toLowerCase();
+      decoded = iconv.decode( data, encoding );
+    } catch ( e ) {
+      cb( e );
```

Comment:
This should probably be `return cb( e )`, otherwise the `cb` will be executed twice.